OKTA-542408 API documentation for CIBA (#3709)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/apps/index.md

| refresh_token <ApiLifecycle access="ea" />                                                                | Refresh token configuration                                                                                                                                                                                                | [Refresh Token object](#refresh-token-object)                                                                                            | TRUE       | FALSE    | TRUE      |
| jwks_uri <ApiLifecycle access="ea" />                                                                     | URL string that references a [JSON Web Key Set](https://tools.ietf.org/html/rfc7517#section-5) for validating JWTs presented to Okta.                                                                                      | String                                                                                        | TRUE       | FALSE    | TRUE      |

> | backchannel_token_delivery_mode <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />                 | The delivery mode for Client-Initiated Backchannel Authentication. Valid types include `poll`, `ping` and `push`, but only `poll` is currently supported.                                                                  | String                                                                                        | TRUE       | FALSE    | TRUE      |

Reviewer: ```suggestion
| backchannel_token_delivery_mode <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />                 | The delivery mode for Client-Initiated Backchannel Authentication. Valid types include `poll`, `ping`, and `push`. Supported values: `poll`.                                                                  | String                                                                                        | TRUE       | FALSE    | TRUE      |
```

Author: updated

> | backchannel_authentication_request_signing_alg <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />  | The signing algorithm for Client-Initiated Backchannel Authentication signed requests. Signed requests will not be supported if this value is not set.                                                                     | String                                                                                        | TRUE       | FALSE    | TRUE      |

Reviewer: When you say "aren't supported" does that mean that an error is sent and the request fails? Or does it just ignore that value?
```suggestion
| backchannel_authentication_request_signing_alg <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />  | The signing algorithm for Client-Initiated Backchannel Authentication signed requests. If this value isn't set, signed requests aren't supported.                                                                     | String                                                                                        | TRUE       | FALSE    | TRUE      |
```

Author: If backchannel_authentication_request_signing_alg is not set and we sent a signed request, the request fails.

Reviewer: ok - suggest the updated edit based on your response.

Author: @susanharper-okta The request will fail only in case we send signed request and backchannel_authentication_request_signing_alg is not set.
If we dont have backchannel_authentication_request_signing_alg set and we send an unsigned request, it would work fine.

Author: updated

> | backchannel_custom_authenticator_id <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />             | The id of the Custom Authenticator that will authenticate the user.                                                                                                                                                        | String                                                                                        | TRUE       | FALSE    | TRUE      |

Reviewer: ```suggestion
| backchannel_custom_authenticator_id <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />             | The ID of the custom authenticator that authenticates the user.                                                                                                                                                        | String                                                                                        | TRUE       | FALSE    | TRUE      |
```

Author: updated

> **Note:** The `refresh_token` <ApiLifecycle access="ea" /> parameter is visible only if the client has `refresh_token` defined as one of its allowed `grant_types`. See [Refresh token object](#refresh-token-object).

> > **Note:** The parameters `backchannel_token_delivery_mode`, `backchannel_authentication_request_signing_alg` and `backchannel_custom_authenticator_id` <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" /> is visible only if the client has `urn:openid:params:grant-type:ciba` defined as one of its allowed `grant_types`.

Reviewer: ```suggestion
> **Note:** The parameters `backchannel_token_delivery_mode`, `backchannel_authentication_request_signing_alg`, and `backchannel_custom_authenticator_id` appear only if the client has `urn:openid:params:grant-type:ciba` defined as one of its allowed `grant_types`. <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />
```

Author: updated
